enarx-pr-request-event

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import enarxbot
+import json
+import os
+import random
+
+QUERY = """
+query($user:String!) {
+  user(login:$user) {
+    status {
+      indicatesLimitedAvailability
+    }
+  }
+}
+"""
+
+# Moves random users from src to dst until src is exhausted or dst has max users.
+#
+# Randomly selected users who have set their GitHub status to `busy` are removed
+# from src but are not added to dst.
+def migrate(dst, src, max=None):
+    while len(src) > 0 and (max is None or len(dst) < max):
+        one = random.sample(src, 1)[0]
+        src.remove(one)
+
+        status = enarxbot.graphql(QUERY, user=one)["user"]["status"]
+        if status is None or not status["indicatesLimitedAvailability"]:
+            dst.add(one)
+
+github = enarxbot.connect()
+
+# Set up metadata
+repo = github.get_repo(os.environ.get('GITHUB_REPOSITORY', None))
+path = os.environ.get('GITHUB_EVENT_PAYLOAD', None)
+event = json.load(path)
+
+# Get reviews team
+org = github.get_organization("enarx")
+reviews = {t.name: t for t in org.get_teams()}['reviews']
+potential_reviewers = {r.login for r in reviews.get_members()}
+
+# Auto-assign for all PRs in the org.
+pr_number = event["pull_request_target"]["number"]
```

Comment:
I don't have access to a good reference for what the `GITHUB_EVENT` payload looks like, so this is an educated guess. I'll need to do some testing when possible in order to figure out if this is correct.